pkg/apis/pipeline/v1beta1/pipeline_validation.go

```diff
@@ -247,6 +247,10 @@ func validateExecutionStatusVariablesExpressions(expressions []string, ptNames s
 		for _, expression := range expressions {
 			// check if it contains context variable accessing execution status - $(tasks.taskname.status)
```

Comment:
need to update this comment to include `$(tasks.status)` as well

Reply:
thanks for the alternative recommendation, its much cleaner 🙏